src/clj_docker_client/utils.clj

```diff
@@ -82,18 +82,12 @@
 
 (defn port-configs-from
   [port-mapping]
-  (let [host-ports      (->> port-mapping
-                             (keys)
-                             (map str))
-        container-ports (->> port-mapping
-                             (vals)
-                             (map str))
-        port-bindings   (for [host-port      host-ports
-                              container-port container-ports]
-                          (PortBinding. (Ports$Binding/parse host-port)
-                                        (ExposedPort/parse container-port)))]
+  (let [port-bindings (for [[host-port container-port] port-mapping
```

Comment:
The `host-port` and `container-port` pairs can be deconstructed from the `port-mapping` map